src/Benchotron/Utils.purs

```diff
@@ -13,5 +13,7 @@ toAny = mkExists <<< Identity
 foreign import unsafeJsonStringify ::
   forall a. a -> String
 
-(>>) :: forall m a b. (Bind m) => m a -> m b -> m b
-(>>) x y = x >>= const y
+bindConst :: forall m a b. (Bind m) => m a -> m b -> m b
```

Comment:
Sorry to nitpick, but I still would rather just get rid of this function entirely and replace uses of it with `*>`.